docs/pages/docs/core-concepts/caching.mdx

```diff
@@ -262,14 +259,14 @@ Not only does `turbo` cache the output of your tasks, it also records the termin
 
 By now, you're probably wondering how `turbo` decides what constitutes a cache hit vs. miss for a given task. Good question!
 
-First `turbo` constructs a hash of the current global state of the monorepo:
+First, `turbo` constructs a hash of the current global state of the monorepo:
 
 - The contents of any files that satisfy the glob patterns and any the values of environment variables listed in [`globalDependencies`](../reference/configuration#globalDependencies-1)
-- The sorted list environment variable key-value pairs that includes `THASH` _anywhere_ in their names (e.g. `STRIPE_PUBLIC_THASH_SECRET_KEY` but not `STRIPE_PUBLIC_KEY`)
+- The sorted list environment variable key-value pairs that includes `THASH` _anywhere_ in their names (e.g. `STRIPE_PUBLIC_THASH_SECRET_KEY`, but not `STRIPE_PUBLIC_KEY`)
 
-Then it adds on more factors relative to a given package's task:
+Then it adds more factors relevant to a given package's task:
 
-- Hash the contents of all not-gitignored files in the package folder or the files matching the `inputs` globs, if present
+- Hash the contents of all version-controlled files in the package folder or the files matching the `inputs` globs, if present
```

Comment:
Some feedback from @mirshko in https://github.com/orgs/vercel/discussions/780#discussioncomment-3409321, that we could probably incorporate into this section while I'm here